docs/migrating/migrating-to-rspack.md

```
@@ -0,0 +1,381 @@
+          runtime: 'classic', // Use 'classic' instead of 'automatic'
+          refresh: env.isDevelopment && env.runningWebpackDevServer,
+        },
+      },
+    },
+  },
+};
+```
+
+**Why?** React on Rails SSR detects render function signatures. The automatic runtime's transformed output may not be detected correctly, causing errors like:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)